spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/inbound/Mqttv5PahoMessageDrivenChannelAdapter.java

```diff
@@ -336,8 +339,10 @@ public void addTopic(String topic, int qos) {
 				this.subscriptions.add(subscription);
 			}
 			if (this.mqttClient != null && this.mqttClient.isConnected()) {
+				MqttProperties subsProperties = new MqttProperties();
```

Comment:
And this one name, please, as `subscriptionProperties`.

Reply:
Fixed.